views/download_calyptia_fluentd.erb

```diff
@@ -0,0 +1,135 @@
+<!--=== Breadcrumbs ===-->
+<div class="breadcrumbs margin-bottom-40">
+  <div class="container">
+    <h1 class="pull-left">Download Fluentd</h1>
+  </div><!--/container-->
+</div><!--/breadcrumbs-->
+<!--=== End Breadcrumbs ===-->
+
+<!--=== Content Part ===-->
+<div class="container">
+  <div class="row portfolio-item margin-bottom-20">
+    <!-- Content Info -->
+    <!--
+    <div class="col-md-12 md-margin-bottom-20">
+    </div>
+    -->
+  </div><!--/row-->
+
+  <div class="row margin-bottom-20">
+    <div class="col-md-9">
+      <h3>Fluentd (v1, current stable)</h3>
+      <div>Fluentd v1 is available on Linux, Mac OSX and Windows. It's packaged by <a href="https://calyptia.com/">Calyptia</a> and <a href="https://www.treasuredata.com/">Treasure Data</a> respectively as:</div>
+      <ul>
+        <li><a href="/download/calyptia_fluentd">Calyptia Fluentd (calyptia-fluentd)</a></li>
+        <li><a href="/download/td_agent">Treasure Agent (td-agent)</a></li>
+        <!-- <li><a href="/download/fluent_package">Fluentd Project (fluent-package)</a></li> -->
+      </ul>
+      <div>In addition, <a href="/download#other-distributions">Docker images, Kubernetes DaemonSet and Ruby gems</a> are also available from the community. </div>
+      <div>All packaged versions are listed in the tables below.</div>
+      <br/>
+      <h4><a name=calyptia-fluentd></a>Calyptia Fluentd</h4>
```

Comment:
I think it can be simplified as follows.
It would be easier to understand and maintain.

```suggestion
      <h3><a name=calyptia-fluentd></a>Calyptia Fluentd</h3>
```